src/core/print.ml

```diff
@@ -8,6 +8,16 @@
 open Timed
 open Extra
 open Terms
+open Console
+open Syntax
+open Sig_state
+
+(** Logging function for printing. *)
+let log_prnt = new_logger 'p' "prnt" "pretty-printing"
+let log_prnt = log_prnt.logger
+
```

Comment:
This does not seem to be used.

Reply:
It has been used and may be useful later as well.